policy/README.md

```diff
@@ -0,0 +1,296 @@
+| `start_time`    | time                        | O   | Beginning time-of-day (hh:mm:ss) when the rule is in effect (default 00:00:00)                                                                                                                                                                                                                                                                          |
+| `end_time`      | time                        | O   | Ending time-of-day (hh:mm:ss) when the rule is in effect (default 23:59:59)                                                                                                                                                                                                                                                                             |
+| `days`          | day[]                       | O   | Days `["sun", "mon", "tue", "wed", "thu", "fri", "sat"]` when the rule is in effect (default all)                                                                                                                                                                                                                                                       |
+| `messages`      | { string:string }           | O   | Message to rider user, if desired, in various languages, keyed by language tag (see [Messages](#messages))                                                                                                                                                                                                                                              |
+| `value_url`     | URL                         | O   | URL to an API endpoint that can provide dynamic information for the measured value (see [Value URL](#value-url))                                                                                                                                                                                                                                        |
+
+### Order of Operations
+
+Rules are ordered most-specific to most-general. E.g. an “earlier” cap rule would take precedence over a “later” cap rule. The internal mechanics of ordering are up to the Policy editing and hosting software.
+
+#### Rule Ordering
+
+Rules, being in a list, are implicitly ordered according to the JSON Specification. Rules are a very specific form of pattern matching; you specify the conditions for which a given rule is 'met', and a vehicle (or series of vehicles) may match with that specific rule. If a vehicle is matched with a rule, then it _will not_ be considered in the subsequent evaluation of rules within a given policy. This allows for expressing complex policies, such as a layer of 'valid' geographies in an earlier rule, with overarching 'invalid' geographies in later rules: see [LADOT Venice Beach Special Operations Example](./Examples.md#venice-beach-spec-ops)
```

Comment:
This describes how rules are prioritized within a given policy, but what happens when rules belonging to different policies collide with each other? It doesn't seem like there's a way for the server to control the resolution of such conflicts since policies are returned in time order instead of priority order.